### ISSUE TITLE
Investigate accessibility of web apps

### ISSUE THREAD

**Issue by d47853** [gchq/gaffer-tools#364]
Ensure that both the UI and Swagger are usable by all.

**Comment by p013570** [comment]
Work ongoing.

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
